packages/theming/atlas/tests/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "../../../../configs/e2e/tsconfig.json",
+    "compilerOptions": {
+        "baseUrl": ".",
+        "module": "commonjs",
+        "lib": ["es2015"],
+        "target": "es2015",
```

Comment:
Could es6 also be a target, then we don't need to specify it here.

Reply:
Sure why not